docs/user_guide/assessment.rst

```diff
@@ -382,13 +384,13 @@ demographic parity may be different when calculated at an aggregate level and
 within more granular categories. In the case of demographic parity, we might 
 need to review 
 :math:`\E[h(X) \given A=a, D=d] = \E[h(X) \given D=d] \quad \forall a` 
-where D represents the feature(s) within X across which members of the groups 
-within A are distributed. 
+where :math:`D` represents the feature(s) within :math:`X`` across which
+members of the groups within :math:`A` are distributed. 
 Demographic parity would then require that the prediction of the target  
 variable is statistically independent of sensitive attributes conditional 
 on D. Simply aggregating outcomes across high-level categories can be 
 misleading when the data can be further disaggregated. 
-It's important to review metrics across these more graular categories, 
+It's important to review metrics across these more granular categories, 
 if they exist, to verify that disparate outcomes persist across all levels 
 of aggregation. 
```

Comment:
Suggest adding a link to the upcoming section on _Intersecting Groups_ during the discussion about considerations for group fairness. Something like "The section on :ref:`intersecting_groups` provides more detail on how to evaluate metrics across intersecting groups."

Reply:
Done.